junifer/testing/datagrabbers.py

```diff
@@ -236,12 +239,14 @@ def get_item(self, subject: str) -> Dict[str, Dict]:
         """
         out = {}
         i_sub = int(subject.split("-")[1]) - 1
-        out["BOLD"] = {"path": Path(self._dataset["func"][i_sub])}
-        conf_format = "fmriprep"
-
+        out["BOLD"] = {
+            "path": Path(self._dataset["func"][i_sub]),
+            "space": "MNI",
```

Comment:
I'd rather use the right space here, so we test correctly the masks/parecellations.